### PR TITLE
added_div_card

### LINKED ISSUE #54: Update result div in index.html
**index.html**
You need to show the following data inside the div(id=result_container), Don't make table with 3 columns as made in any of the designs.
**Show some creativity** 
ONLY MAKE CHANGES IN div#result_container
NOTE: while working add hidden class to div with id=landing_container and input_container
_while submitting add hidden class to input_container and result container_
If there is any confusion, kindly ask : ).

For now hard code this data

Disease Name: Pepper__bell___Bacterial_spot
Symptoms : 
1 : Leaf spots that appear on the lower surface of older leaves as small, pimples and on the upper leaf surface as small water-soaked spots are a symptom of bacterial spot.
2 : Eventually, the spots develop gray to tan centers with darker borders. Lesions enlarge during warm, humid weather. Leaves may then turn yellow, then brown and drop. Lesions may also develop on stems.
3:  Lesions may also develop on stems. Fruits develop small, raised rough spots that do not affect eating quality. Severely infected leaves will drop resulting in

Treatment :
1 : Treat seeds by soaking them for 2 minutes in a 10% chlorine bleach solution (1 part bleach; 9 parts water). Thoroughly rinse seeds and dry them before planting.
2 : Mulch plants deeply with a thick organic material like newspaper covered with straw or grass clippings.               
3 : Spray every 10-14 days with fixed copper (organic fungicide) to slow down the spread of infection.
4: Rotate peppers to a different location if infections are severe and cover the soil with black plastic mulch or black landscape fabric prior to planting.                         
                      
             
Recommended products :
1 : Neem Oil : https://www.planetnatural.com/product/organic-neem-oil/
2 : Garden Dust : https://www.planetnatural.com/product/garden-dust/

### PR DESCRIPTION
**Pull request by abhishekgupta368**
Update result div in index.html #54

file added:
-> modified index.html

feature implemented:
-> create the card like view for presenting the leaf

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -24,21 +24,48 @@
...
-    <div id="background">
-      <div id="landing_container" >
+    <div id="background" class = "clear">
+
+      <div id="landing_container" class="hidden">
         <h1 class="heading">CROP AI</h1>
         <div class="button-container">
           <a href="#" class="button-Index">ABOUT</a>
-          <a href="#" class="button-Index" onclick="show_input_container()">ANALYZE</a>
+          <a href="#" class="button-Index">ANALYZE</a>
```
Comment (rajats98) [inline src/index.html]: why did you remove onclick ?
let it be there

**Review by rajats98 — CHANGES_REQUESTED** [review]
hey,
Please make the box height self-adjusting, if content is less its height should be less, and if content is more the height should increase. (the text should not go outside, the height should adjust itself)

also make sure when height increases, it should not hide behind the bottom leaves. 

increase font size at least by 4px, you may increase width the of box.

over-all design is good

**Comment by rajats98** [comment]
Also, don't use \<br> tags in between of one line, the text should auto wrap, it should not go outside the block  

For now, we are adding dummy data, In actual, the data will come from API and then we can't add <br> tag at relevant position

**Comment by rajats98** [comment]
Also add 4th point of treatment, as given in issue description

**Comment by abhishekgupta368** [comment]
can you please make point short, please. I am not able to fit big points on the card, other wise the font-size will become small.

**Comment by rajats98** [comment]
> can you please make point short, please. I am not able to fit big points on the card, other wise the font-size will become small.

point can't be made short, actually, data will come from API... and the data I have given, is a real possibilty.
 
try increasing font by 2px, and increase the width, don't worry if the page become scrollable, in case it is required.

also keep in mind that the data may contain only 1-2 points under each heading or It may contain 4-5 points, so make your code flexible to adapt itself.

it should not happen that you set a very large height to box, or it should also not happen that your text exceeds the box, both give really bad experience to user.

Take this as a challenge, this is very basic design practice

**Comment by rajats98** [comment]
do a little bit of experimentation and research, do it... it will also help you in future, this is a very common use-case when you develop a UI

**Comment by rajats98** [comment]
I have updated index.css it may help you :) pull the latest code

File: src/index.html [hunk 2]
```diff
@@ -32,20 +32,48 @@ <h1 class="heading">CROP AI</h1>
...
 
       <div id="input_container" class="hidden"></div>
-      
-      <div id="result_container" class="hidden"></div>
-    
+
+      <div id="result_container">
+        <div id="box">
+          <div id="float-child">
+            <img id= "pepper-bell" src="./img/leaf.jfif" alt="Image not found"  height= 300px width="200px">
```
Comment (rajats98) [inline src/index.html]: change id to crop_img (every time, it will not be bell pepper)

Add hidden class to landing_container (because only one div will be shown at one time, so develop it with that mindset)

File: src/assets/index.css [hunk 3]
```diff
@@ -101,3 +102,29 @@ padding: 2%;
...
+    float: left;
+    padding: 20px;
+    height: 100%;
+
+}
+#list_head{
+  text-decoration: underline;
+  font-weight: bold;
+}
+#pepper-bell{
```
Comment (rajats98) [inline src/assets/index.css]: remove this, if no css is added

File: src/index.html [hunk 4]
```diff
@@ -32,20 +32,48 @@ <h1 class="heading">CROP AI</h1>
...
+          </div>
+          <div id="float-child">
+            <div id="heading">
+              <h3>Pepper Bell Bacterial Spots </h3>
+            </div>
+            <p id="list_head">Symptoms</p>
```
Comment (rajats98) [inline src/index.html]: id should be unique and should be used for one element, you want use class (list_head, symptoms)
change symptoms to a more meaningful class like list_content

**Review by rajats98 — CHANGES_REQUESTED** [review]

**Comment by rajats98** [comment]
@abhishekgupta368 make sure you read all reviews and all the instructions properly before making updating PR
Read the instructions one more, go through all the review above.

When the landing page is hidden it gives a very much unaligned, poorly position page, 
and none of the reviews have been incorporated

File: src/index.html (hunk 2 above)
Comment (abhishekgupta368) [reply src/index.html]: done

**Comment by rajats98** [comment]
Great, 
Now some extra task for you :fist_oncoming: 
add this tag in index.css (below` <meta charset="utf-8">`)
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`
<br> <br>
- do this for div with class **`.leaf-class2 `**
1. css add these properties
    ```
    z-index: -10;
    opacity:80%;
    ```
1. remove height attribute from tag, change width="80%"
<br> <br>
- do this for div with class **`.leaf-class1 `** 
1. css add these properties
     ```
    position:fixed;
    right:0;
    z-index: -10;
    opacity:80%;
   ```
1. remove height and width attribute from tag

**Comment by abhishekgupta368** [comment]
which tag

**Comment by rajats98** [comment]
hey updated,
edited the prev comment,
